accounts/models.py

```diff
@@ -6,6 +6,8 @@ class Account(models.Model):
     first_name = models.CharField(max_length=50, blank=False)
     last_name = models.CharField(max_length=50, blank=True)
     mobile_number = models.CharField(max_length=12, blank=True)
+    profile = models.URLField(blank=True)
```

Comment:
rename it to `profile_pic` or `image_url`

Reply:
profile field has been renamed to profile_pic in account model changes are reflected in the recent push.